progressbar/progressbar_test.go

```diff
@@ -0,0 +1,89 @@
+package progressbar
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandProgress_InitializeProgressBar(t *testing.T) {
+	t.Parallel()
+	cp := &CommandProgressBar{}
+	cp.InitializeProgressBar()
+	assert.Equal(t, "*pb.ProgressBar", reflect.TypeOf(cp.progressbar).String())
+}
+
+func TestCommandProgress_Finish(t *testing.T) {
+	t.Parallel()
+	cp := &CommandProgressBar{}
+	cp.InitializeProgressBar()
+
+	cp.Finish()
+
+	assert.Equal(t, true, cp.progressbar.IsFinished())
+}
+
+func TestCommandProgress_IncrementCompletedObjects(t *testing.T) {
+	t.Parallel()
+	cp := &CommandProgressBar{}
+	cp.InitializeProgressBar()
+	cp.IncrementCompletedObjects()
+	assert.Equal(t, int64(1), cp.completedObjects)
+	// Verify that the progress bar's task status has been updated
+	assert.Equal(t, true, strings.Contains(cp.progressbar.String(), "1/0"))
```

Comment:
Actually, the rendering template is defined by us in `progressbarTemplate`. `{{ string . "objects" | yellow}}` and `cp.progressbar.Set("objects", fmt.Sprintf("(%d/%d)", cp.completedObjects, cp.totalObjects))` defines how we render the number of objects. It is not a feature of the library, so I don't think it will be broken due to an update in the library.